Snowflake.Data/Client/SnowflakeDbConnectionPool.cs

```diff
@@ -129,6 +130,9 @@ internal static void SetConnectionPoolVersion(ConnectionPoolType requestedPoolTy
         {
             lock (s_connectionManagerInstanceLock)
             {
+                if (s_connectionManager != null)
```

Comment:
it would disable the possibility to switch to the old pool

Reply:
Fixed